datastore/src/test/java/io/spine/server/storage/datastore/DatastoreWrapperTest.java

```diff
@@ -308,34 +306,37 @@ void testIaeOnBatchReadWithZeroSize() {
     @Test
     @DisplayName("generate key factories aware of tenancy")
     void testGenerateKeyFactory() {
-        DatastoreWrapper wrapper = wrap(localDatastore(), multitenant());
-        String tenantId1 = "first-tenant-ID";
-        String tenantId1Prefixed = "Vfirst-tenant-ID";
-        String tenantId2 = "second@tenant.id";
-        String tenantId2Prefixed = "Esecond-at-tenant.id";
-        String tenantId3 = "third.id";
-        String tenantId3Prefixed = "Dthird.id";
-        Datastore datastore = wrapper.datastore();
+        var wrapper = wrap(localDatastore(), multitenant());
+        var tenantId1 = "first-tenant-ID";
+        var tenantId1Prefixed = "Vfirst-tenant-ID";
+        var tenantId2 = "second@tenant.id";
+        var tenantId2Prefixed = "Esecond-at-tenant.id";
+        var tenantId3 = "third.id";
+        var tenantId3Prefixed = "Dthird.id";
+        var datastore = wrapper.datastore();
         ensureNamespace(tenantId1Prefixed, datastore);
         ensureNamespace(tenantId2Prefixed, datastore);
         ensureNamespace(tenantId3Prefixed, datastore);
-        EmailAddress emailAddress2 = EmailAddress
+        var emailAddress2 = EmailAddress
                 .newBuilder()
                 .setValue(tenantId2)
                 .vBuild();
-        InternetDomain internetDomain3 = InternetDomain
+        var internetDomain3 = InternetDomain
                 .newBuilder()
                 .setValue(tenantId3)
                 .vBuild();
-        TenantId id1 = TenantId.newBuilder()
-                               .setValue(tenantId1)
-                               .vBuild();
-        TenantId id2 = TenantId.newBuilder()
-                               .setEmail(emailAddress2)
-                               .vBuild();
-        TenantId id3 = TenantId.newBuilder()
-                               .setDomain(internetDomain3)
-                               .vBuild();
+        var id1 = TenantId
+                .newBuilder()
```

Comment:
I think, our current convention dictates that the `newBuilder()` was on the same line with the type name.